controllers/admin/AdminSearchController.php

```diff
@@ -195,6 +195,7 @@ public function postProcess()
             if (!$searchType || $searchType == 7) {
                 /* Handle module name */
                 if ($searchType == 7 && Validate::isModuleName($this->query) && ($module = Module::getInstanceByName($this->query)) && Validate::isLoadedObject($module)) {
+                    // @todo redirect directly to module manager with search prefilled, because this won't work anymore
```

Comment:
Please create an issue. Not anymore todo without linked issue.

Reply:
https://github.com/PrestaShop/PrestaShop/issues/28303